packages/edit-site/src/components/sidebar-navigation-screen-page/style.scss

```diff
@@ -0,0 +1,66 @@
+.edit-site-sidebar-navigation-screen-page {
+	.edit-site-sidebar-navigation-screen__sticky-section {
+		padding: $grid-unit-40 0;
+		margin: $grid-unit-40 0;
+	}
+}
+
+.edit-site-sidebar-navigation-screen-page__featured-image-wrapper {
+	padding: $grid-unit-10;
+	background-color: $gray-800;
+	margin-bottom: $grid-unit-20;
+	min-height: 128px;
+	.edit-site-sidebar-navigation-screen-page__featured-image {
```

Comment:
Similar nit to another comment, is the nesting here needed?

Reply:
Not here! 🙇